tools/src/main/scala/scala/scalanative/build/Build.scala

```diff
@@ -24,7 +24,9 @@ object Build {
    *  val clangpp  = Discover.clangpp()
    *  val linkopts = Discover.linkingOptions()
    *  val compopts = Discover.compileOptions()
-   *  val triple   = Discover.targetTriple(clang, workdir)
+   *  // Empty string or target triple if cross compiling
+   *  val triple   = ""
```

Comment:
Given my other comment, this shouldn't be an empty string. It can be `None`, or `Some(actualTriple)`, or `actualTriple`.

Reply:
It is not required so I just removed it from the documentation altogether.